WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -679,6 +703,51 @@ - (void)configureTodaysStatsCell:(InsightsTodaysStatsTableViewCell *)cell
 
 }
 
+- (void)configureSectionSelectableCell:(StatsSelectableTableViewCell *)cell forRow:(NSInteger)row
+{
+    cell.selectedIsLighter = NO;
+    cell.unselectedCellValueColor = [WPStyleGuide wordPressBlue];
+    
+    StatsSummary *todaySummary = self.sectionData[@(StatsSectionInsightsTodaysStats)];
+    
+    switch (row) {
+        case 1: // Views
+        {
+            cell.categoryIconLabel.text = @"";
+            cell.categoryLabel.text = [NSLocalizedString(@"Views", @"") uppercaseStringWithLocale:[NSLocale currentLocale]];
```

Comment:
Maybe we could use a category method... `uppercaseLocaleString` perhaps! (Just thinking aloud, no need to do anything!)